docs/docs/stylesheets/extra.css

```diff
@@ -84,7 +85,13 @@ Fonts are the same as pkgdown websites (otherwise tables not properly formatted)
 	scrollbar-color: var(--custom-scrollbar-col);
 }
 
+.md-top:hover {
+	background: var(--md-typeset-a-color-hover);
```

Comment:
It looks like tabs and spaces are mixed in this file.

Reply:
It all appears the same to me in RStudio, do you know an automatic styler for CSS files?

Reply:
> It all appears the same to me in RStudio, do you know an automatic styler for CSS files?

Maybe [Prettier](https://prettier.io/)?

I believe the popular method is to specify the tab size with Editorconfig; I don't think RStudio IDE supported Editorconfig yet.